<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>616</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTreeView" name="gitTree">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>221</width>
      <height>561</height>
     </rect>
    </property>
   </widget>
   <widget class="QPlainTextEdit" name="commitMsg">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>0</y>
      <width>271</width>
      <height>151</height>
     </rect>
    </property>
   </widget>
   <widget class="QTextBrowser" name="workTree">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>0</y>
      <width>311</width>
      <height>341</height>
     </rect>
    </property>
   </widget>
   <widget class="QTextBrowser" name="showDiff">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>340</y>
      <width>581</width>
      <height>221</height>
     </rect>
    </property>
   </widget>
   <widget class="QScrollBar" name="gitTreeHorizontalScrollBar">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>540</y>
      <width>191</width>
      <height>20</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QScrollBar" name="gitTreeVerticalScrollBar">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>10</y>
      <width>20</width>
      <height>531</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="QScrollBar" name="horizontalScrollBar_2">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>320</y>
      <width>281</width>
      <height>20</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QScrollBar" name="verticalScrollBar_2">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>10</y>
      <width>20</width>
      <height>311</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="QScrollBar" name="horizontalScrollBar_3">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>540</y>
      <width>551</width>
      <height>20</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QScrollBar" name="verticalScrollBar_3">
    <property name="geometry">
     <rect>
      <x>780</x>
      <y>350</y>
      <width>20</width>
      <height>191</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="QScrollBar" name="horizontalScrollBar_4">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>130</y>
      <width>241</width>
      <height>20</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QScrollBar" name="verticalScrollBar_4">
    <property name="geometry">
     <rect>
      <x>780</x>
      <y>10</y>
      <width>20</width>
      <height>121</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="QPushButton" name="addMsg">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>150</y>
      <width>271</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>Push</string>
    </property>
   </widget>
   <widget class="QTextBrowser" name="gitLogMsg">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>180</y>
      <width>271</width>
      <height>161</height>
     </rect>
    </property>
   </widget>
   <widget class="QScrollBar" name="horizontalScrollBar_5">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>320</y>
      <width>241</width>
      <height>20</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QScrollBar" name="verticalScrollBar_5">
    <property name="geometry">
     <rect>
      <x>780</x>
      <y>190</y>
      <width>20</width>
      <height>131</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>26</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionNew"/>
    <addaction name="actionOpen_Git_Repository"/>
    <addaction name="actionGit_Config"/>
    <addaction name="actionExecute_Command"/>
   </widget>
   <widget class="QMenu" name="menuExit">
    <property name="title">
     <string>Exit</string>
    </property>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuCode">
    <property name="title">
     <string>Code</string>
    </property>
    <addaction name="actionPull"/>
    <addaction name="actionPush"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuCode"/>
   <addaction name="menuExit"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionNew">
   <property name="text">
    <string>New Git Repository</string>
   </property>
  </action>
  <action name="actionOpen_Git_Repository">
   <property name="text">
    <string>Open Git Repository</string>
   </property>
  </action>
  <action name="actionGit_Config">
   <property name="text">
    <string>Git Config</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
  <action name="actionExecute_Command">
   <property name="text">
    <string>Execute Command</string>
   </property>
  </action>
  <action name="actionPull">
   <property name="text">
    <string>Pull</string>
   </property>
  </action>
  <action name="actionPush">
   <property name="text">
    <string>Push</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
